dev-tools/packer/platforms/darwin/build.sh

```diff
@@ -17,3 +17,6 @@ docker run --rm -v ${BUILD_DIR}:/build \
     tudorg/fpm /build/run-$runid.sh
 
 rm ${BUILD_DIR}/settings-$runid.yml ${BUILD_DIR}/run-$runid.sh
+
+echo 'Running macOS packager'
+${BASEDIR}/pkg/build.sh
```

Comment:
I think there should be a conditional preventing this from error'ing out if we aren't on Darwin.

Or it might be better to separate this into it's own build target such that you can run `make packages` on any machine with Docker and can run `make macos-dmgs` (naming TBD) afterwards to build the the DMGs.